input-explainer.md

```diff
@@ -176,16 +176,30 @@ Most applications will want to visually represent the input sources somehow. The
   * `'tracked-pointer'`: If the `gripTransform` is not `null` an application-appropriate controller model should be drawn using that transform. If appropriate for the experience, the a visualization of the target ray and a cursor as described in the `'gaze'` should also be drawn.
   * `'screen'`: In all cases the point of origin of the target ray is obvious and no visualization is needed.
 
+Determining what to render is important as well. In some cases it's appropriate for the application to render a contextually appropriate model (such as a racket in a tennis game), in which case the physical appearance of the input source doesn't matter much. Many other times, though, it's desirable for the application to display a device that matches what the user is holding, especially when relaying instructions about it's use. Finally there are times when it's best to not render anything at all, such as when the XR device uses a transparent display and the user can see their hands and/or any tracked devices without app assistance.
+
+ The `XRInputSource`'s `renderId` is used to determine what should be rendered in the cases when an application specific model isn't being used. 
```

Comment:
Even if the app has its own application-specific models for the most popular controllers out there, it still needs this ID to select between them.

It's perhaps more correct to say "determine what should be rendered if the app intends to visualize the input source itself, rather than an alternative virtual object.